static/app/components/dropdownMenu.tsx

```diff
@@ -135,6 +129,12 @@ class DropdownMenu extends React.Component<Props, State> {
     document.removeEventListener('click', this.checkClickOutside, true);
   }
 
+  dropdownMenu: Element | null = null;
```

Comment:
There was a new warning rule that want's these to be moved down below `componentWillUnmount`